applications/social-networks.md

```diff
@@ -0,0 +1,17 @@
+---
+description: Machine translation for social media
+---
+
+# Social Networks
+
+Machine translation for **social networks** such as Facebook or Twitter is the translation of specific content chosen by the reader. The machine translation is used for gisting.
+
+While the static content of social networks is human-translated, machine translation is used for **user-generated content**.
+
+### Content types
+
+- Comments
+- Reviews
+- Posts
+
+In YouTube, machine translation is used in automatic closed captions.
```

Comment:
On YouTube, machine translation is for subtitles, including autogenerated subtitles.